src/Datadog.Trace.OpenTracing/OpenTracingTracer.cs

```diff
@@ -53,7 +53,7 @@ public ISpanBuilder BuildSpan(string operationName)
                 return codec.Extract(carrier);
             }
 
-            throw new NotSupportedException($"Tracer.Extract is not implemented for {format} by Datadog.Trace");
+            throw new NotSupportedException($"Tracer.Extract is not implemented for {format} by OpenTelemetry.Instrumentation");
```

Comment:
IMO this message reflects the type name. I think it should not be changed

Reply:
This is done according to https://github.com/signalfx/signalfx-dotnet-tracing/pull/31/files
and I think in this case this is name of the assembly, not the type.

Reply:
You are correct